src/core_plugins/timelion/public/chart.less

```diff
@@ -11,6 +11,17 @@
   overflow: hidden;
 }
 
+div.legend table caption {
```

Comment:
Could we use a class name instead of using element names, to align it more with out CSS style guide?